rtc_exec_enclave/src/types.rs

```diff
@@ -0,0 +1,13 @@
+use std::boxed::Box;
+
+pub(crate) type CallReturnValue = Box<[u8]>;
+
+pub(crate) type ExecResult = core::result::Result<CallReturnValue, ()>;
+
+pub(crate) trait ExecModule {
+    /// Calls the entry function of a module with the provided dataset and return the result
+    ///
+    /// # Safety
+    /// The caller should ensure that `dataset_ptr` is a valid pointer to a `u8` slice of `dataset_len`
```

Comment:
```suggestion
    /// The caller must ensure that `dataset_ptr` is a valid pointer to a `u8` slice of `dataset_len`
```

(for RFC 2119 style?)